servers/src/main/java/tachyon/master/file/meta/InodeTree.java

```diff
@@ -222,10 +222,10 @@ public CreatePathResult createPath(TachyonURI path, CreatePathOptions options)
       // exception here. Otherwise we add the remaining path components to the list of components
       // to create.
       if (!options.isRecursive()) {
-        final String msg = "File " + path + " creation failed. Component "
-            + traversalResult.getNonexistentPathIndex() + "("
-            + parentPath[traversalResult.getNonexistentPathIndex()] + ") does not exist";
-        LOG.info("InvalidPathException: " + msg);
+        final String msg = String.format("File %s creation failed. Component %d(%s) does not exist",
```

Comment:
StringBuilder might be a better choice here:
http://stackoverflow.com/questions/513600/should-i-use-javas-string-format-if-performance-is-important